sdk/compute/azure-mgmt-imagebuilder/azure/mgmt/imagebuilder/models/_models.py

```diff
@@ -539,23 +539,23 @@ def __init__(self, **kwargs):
 
 class ImageTemplatePlatformImageSource(ImageTemplateSource):
     """Describes an image source from [Azure Gallery
-    Images](https://docs.microsoft.com/en-us/rest/api/compute/virtualmachineimages).
+    Images](https://docs.microsoft.com/rest/api/compute/virtualmachineimages).
```

Comment:
Changing generated code has no value, since the next generation will put it back again.
I would suggest to make a list of mgmt packages that have those kind of issues, and open Swagger bugs instead.

For instance, the root cause of this one is here:
https://github.com/Azure/azure-rest-api-specs/blob/900ef530bda427a6ba9f61ac6f64952ea7650a37/specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2020-02-14/imagebuilder.json#L579

Reply:
Thanks i'll make these changes in this PR - https://github.com/Azure/azure-sdk-for-python/pull/12697